test/parallel/test-stream3-pause-then-read.js

```diff
@@ -42,7 +42,7 @@ function read100() {
 function readn(n, then) {
   console.error('read %d', n);
   expectEndingData -= n;
-  ;(function read() {
+  (function read() {
```

Comment:
Right, but since this is node and we operate in modules, where nothing gets exposed to the global scope anyways, isn't that kind of moot?